### ISSUE TITLE
For translation purpose

### ISSUE THREAD

**Issue by popoqc** [cmdrmcdonald/EliteDangerousDataProvider#195]
Hi ,
First I would like to say that I'm using EDDI in french with a french personality. I have edited the translations.cs file to better fit in french too.

So I found that in the Ship.cs file it is also hard-coded "your " and think it would be better to have it in the personality *.json script file instead. Like "Your {ShipName()} ... bla bla"

So the code should look like this : 
`public string SpokenName(string defaultname = null)
        {
            string model = defaultname == null ? SpokenModel() : defaultname;
            if (model == null)
            {
                model = "ship";
            }
            //string result = ("your " + model);
            string result = model;
            if (phoneticname != null)
            {
                result = "<phoneme alphabet=\"ipa\" ph=\"" + phoneticname + "\">" + name + "</phoneme>";
            }
            else if (name != null)
            {
                result = name;
            }
            return result;
        }`

**Comment by cmdrmcdonald** [comment]
Thank you for your suggestion.  The problem with doing this is that ShipName() is designed to only write "your" in front of the ship if it provides a model, e.g. "your Anaconda".  If the ship has a name, e.g. "HMS Victory", then your suggestion would result in EDDI saying "your HMS Victory" rather than just "HMS Victory".

EDDI is not designed for internationalisation, but I'll take a look at this and see what can be done.

**Comment by popoqc** [comment]
Yes I agree with you. I was experiencing it this week-end and you are 
right on this. There should not be any "your" in from of a personalized 
ship name.
